alphalens/utils.py

```diff
@@ -235,7 +235,7 @@ def compute_forward_returns(factor_idx,
         # in between the trades so we'll test several entries to find out the
         # correct period length
         #
-        entries_to_test = min(10, len(fwdret.index)-period)
+        entries_to_test = min(10, len(fwdret.index), len(prices.index)-period)
```

Comment:
was this a bug?

Reply:
That's a check to avoid an "index out of range" exception but the check was too restrictive and the function didn't work on a small amount of data.So I fixed the check